svn diff -r7052:7110 https://gnunet.org/svn/GNUnet/configure.ac
r7110 | amatus | 2008-06-09 18:34:31 +0200 (Mo, 09 Jun 2008) | 2 lines
Added AC_PROG_CXX to make configure work in archlinux.

See also https://bugs.gentoo.org/show_bug.cgi?id=226195

Index: configure.ac
===================================================================
--- configure.ac	(revision 7052)
+++ configure.ac	(revision 7110)
@@ -31,6 +31,7 @@
 AC_PROG_AWK
 AC_PROG_CC
 AC_PROG_CPP
+AC_PROG_CXX
 AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
